lib/src/cartesian/CartesianChart.tsx

```diff
@@ -369,6 +374,28 @@ export function CartesianChart<
      */
     .activateAfterLongPress(gestureLongPressDelay);
 
+  if (chartPressConfig?.pan?.activateAfterLongPress !== undefined) {
+    if (isNaN(chartPressConfig.pan.activateAfterLongPress)) {
+      panGesture.config.activateAfterLongPress = undefined;
```

Comment:
I'm not sure if we need this `isNaN` check do we? Linter seems okay without it and docs/compiler make it clear it needs to be a number.

Reply:
refactored